### PR TITLE
worker: init at 3.8.5

### PR DESCRIPTION
**Pull request by ndowens**
###### Things done
Add worker FM to repo
- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [x] macOS
   - [ ] Linux
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/misc/worker/default.nix [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  url = "http://www.boomerangsworld.de/cms/worker/downloads/${name}-${version}.tar.gz";
+  sha256 = "1xy02jdf60wg2jycinl6682xg4zvphdj80f8xgs26ip45iqgkmvw";
+};
+
+buildInputs = with pkgs; [ xorg.libX11 ];
+
+meta = with stdenv.lib; {
+  description = "a two-pane file manager with advanced file manipulation features";
+  homepage = "http://www.boomerangsworld.de/cms/worker/index.html";
+  licenses = license.gpl2;
```
Comment (LnL7) [inline pkgs/applications/misc/worker/default.nix]: `licenses.gpl2`

File: pkgs/applications/misc/worker/default.nix [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  maintainers = [ maintainers.ndowens ];
```
Comment (LnL7) [inline pkgs/applications/misc/worker/default.nix]: You have to add an entry to [lib/maintainers.nix](https://github.com/NixOS/nixpkgs/blob/master/lib/maintainers.nix)

File: pkgs/applications/misc/worker/default.nix [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  url = "http://www.boomerangsworld.de/cms/worker/downloads/${name}-${version}.tar.gz";
+  sha256 = "1xy02jdf60wg2jycinl6682xg4zvphdj80f8xgs26ip45iqgkmvw";
+};
+
+buildInputs = with pkgs; [ xorg.libX11 ];
+
+meta = with stdenv.lib; {
+  description = "a two-pane file manager with advanced file manipulation features";
+  homepage = "http://www.boomerangsworld.de/cms/worker/index.html";
+  license =  licenses.gpl2;
```
Comment (LnL7) [inline pkgs/applications/misc/worker/default.nix]: If you add `platforms = platforms.unix;` hydra will build this for linux and darwin.

File: lib/maintainers.nix [hunk 4]
```diff
@@ -336,6 +336,7 @@
   Nate-Devv = "Nathan Moore <natedevv@gmail.com>";
   nathan-gs = "Nathan Bijnens <nathan@nathan.gs>";
   nckx = "Tobias Geerinckx-Rice <tobias.geerinckx.rice@gmail.com>";
+  ndowens = "Nathan Owens <ndowens04@gmail.com";
```
Comment (LnL7) [inline lib/maintainers.nix]: not sure, but the `>` might be needed to make notifications from hydra work when something breaks.

File: pkgs/applications/misc/worker/default.nix [hunk 5]
```diff
@@ -0,0 +1,20 @@
+{pkgs, stdenv, xorg, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name = "worker";
+  version = "3.8.5";
+
+src = fetchurl {
```
Comment (LnL7) [inline pkgs/applications/misc/worker/default.nix]: Can you indent this one level, same with the meta section.

**Review by LnL7 — COMMENTED** [review]
Just some minor things

**Comment by LnL7** [comment]
I also tested this on darwin.

File: lib/maintainers.nix (hunk 4 above)
Comment (ndowens) [inline lib/maintainers.nix]: Fixed, left that out by accident :)

File: pkgs/applications/misc/worker/default.nix (hunk 5 above)
Comment (ndowens) [inline pkgs/applications/misc/worker/default.nix]: Indented, hopefully to what it should be now

Comment (LnL7) [inline pkgs/applications/misc/worker/default.nix]: you missed the closing curly braces :)

Comment (ndowens) [inline pkgs/applications/misc/worker/default.nix]: Done

**Pull request merged**